app/.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_BASE_URL=http://localhost:8001
```

Comment:
Great idea about creating the `.env.example` file.

Can you include all the unique keys that are in our `.env` file in `.env.example`? (except the values of course!).

I think we should rename our current `.env` keys as they use lower case key names whereas best practice is upper case. But let's not do this in this task as that would break the rest of the app.